### ISSUE TITLE
Feature Request: Hypernetwork/VAE switching within plugin UI

### ISSUE THREAD

**Issue by JTJutajoh** [isekaidev/stable.art#8]
In addition to the dropdown for choosing models, it would be very helpful to have a matching dropdown for switching Hypernetworks that the AUTOMATIC1111 webui is aware of. The same goes for VAEs.

**Comment by DarkAlchy** [comment]
My drop-down for model is greyed out and yours isn't?

**Comment by JTJutajoh** [comment]
Yeah, you're right. Mine is today too. It was working fine last time I used the plugin

**Comment by DarkAlchy** [comment]
> Yeah, you're right. Mine is today too. It was working fine last time I used the plugin

I just grabbed it and was just wondering.

**Comment by Woisek** [comment]
> In addition to the dropdown for choosing models, it would be very helpful to have a matching dropdown for switching Hypernetworks that the AUTOMATIC1111 webui is aware of. The same goes for VAEs.

It would also make sense to have the embeddings, too.
Please.

**Comment by JTJutajoh** [comment]
> > In addition to the dropdown for choosing models, it would be very helpful to have a matching dropdown for switching Hypernetworks that the AUTOMATIC1111 webui is aware of. The same goes for VAEs.
> 
> It would also make sense to have the embeddings, too. Please.

Embeddings are just included as part of the prompt, you don't really need a dropdown for them.

**Comment by Woisek** [comment]
> Embeddings are just included as part of the prompt, you don't really need a dropdown for them.

So one has to remember what exactly was the correct wording for one out of 40+ embeddings ... :)
Some "click-and-it-copies-to-the-prompt" would be helpful.

**Comment by JTJutajoh** [comment]
> > Embeddings are just included as part of the prompt, you don't really need a dropdown for them.
> 
> So one has to remember what exactly was the correct wording for one out of 60+ embeddings ... :) Some "click-and-it-copies-to-the-prompt" would be helpful.

I don't have nearly that many embeddings, let alone ones that I actually use regularly. But I can see your point if you do have that many.

**Comment by Woisek** [comment]
> I don't have nearly that many embeddings, let alone ones that I actually use regularly. But I can see your point if you do have that many.

Right now it's 65 to be exact. 5 from myself I tried to create.
I have found out, that embeddings are flexible and often produce astounding results compared to their small size. And I read more and more that embeddings are supposed to be even better than hypernetworks, but easier to create. For me a viable way, since I have a graphics card with only 8 GB VRAM. :)

**Comment by JTJutajoh** [comment]
I want to add to this feature request the ability to switch Lora models as well. 

the Webui lets you add things like Loras or Hypernetworks by putting them in the prompt with a special syntax inside angle brackets, it would be nice if the plugin UI could have some sort of list of installed Loras (And the other types like Hypernetworks) and it adds them to the prompt when chosen.

**Comment by isekaidev** [comment]
It is still not a list of Lora/Hypernetwork models, but now you can use them in your prompts.

d759649a0bc86648badc66d80e3e12c2206a44e1
